eslint.config.js

```diff
@@ -0,0 +1,196 @@
+import js from "@eslint/js";
+import ts from "@typescript-eslint/eslint-plugin";
+import tsParser from "@typescript-eslint/parser";
+import { resolve as tsResolver } from "eslint-import-resolver-typescript";
+import importPlugin from "eslint-plugin-import";
+import jsdoc from "eslint-plugin-jsdoc";
+import perfectionist from "eslint-plugin-perfectionist";
+import sonarjs from "eslint-plugin-sonarjs";
+import unicorn from "eslint-plugin-unicorn";
+import globals from "globals";
+
+const JS_MAX_PARAMS_ALLOWED = 3;
+
+/** @typedef {import("eslint").Linter.FlatConfig} */
```

Comment:
```suggestion
/** @typedef {import("eslint").Linter.Config} */
```
I believe FlatConfig is deprecated